Microsoft.Toolkit.Uwp.UI.Controls/OrbitView/OrbitView.cs

```diff
@@ -0,0 +1,687 @@
+            get { return (DoubleCollection)GetValue(OrbitDashArrayProperty); }
+            set { SetValue(OrbitDashArrayProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="OrbitDashArrayProperty"/> property
+        /// </summary>
+        public static readonly DependencyProperty OrbitDashArrayProperty =
+            DependencyProperty.Register("OrbitDashArray", typeof(DoubleCollection), typeof(OrbitView), new PropertyMetadata(null, OnOrbitPropertyChanged));
+
+        /// <summary>
+        /// Gets or sets a value indicating the thickness of the anchors
+        /// </summary>
+        public double AnchorThickness
+        {
+            get { return (double)GetValue(AnchorThicknessProperty); }
+            set { SetValue(AnchorThicknessProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="AnchorThicknessProperty"/> property
+        /// </summary>
+        public static readonly DependencyProperty AnchorThicknessProperty =
+            DependencyProperty.Register("AnchorThickness", typeof(double), typeof(double), new PropertyMetadata(1d, OnAnchorPropertyChanged));
+
+        /// <summary>
+        /// Gets or sets a value indicating the thickness of the orbits
+        /// </summary>
+        public double OrbitThickness
+        {
+            get { return (double)GetValue(OrbitThicknessProperty); }
+            set { SetValue(OrbitThicknessProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="OrbitThicknessProperty"/> property
+        /// </summary>
+        public static readonly DependencyProperty OrbitThicknessProperty =
+            DependencyProperty.Register("OrbitThickness", typeof(double), typeof(OrbitView), new PropertyMetadata(1d, OnOrbitPropertyChanged));
+
+        /// <summary>
+        /// Gets or sets a value representing the center element
+        /// </summary>
+        public object CenterContent
+        {
+            get { return (int)GetValue(CenterContentProperty); }
+            set { SetValue(CenterContentProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="CenterContentProperty"/> property
+        /// </summary>
+        public static readonly DependencyProperty CenterContentProperty =
+            DependencyProperty.Register("CenterContent", typeof(int), typeof(OrbitView), new PropertyMetadata(0));
+
+        protected override DependencyObject GetContainerForItemOverride()
+        {
+            var element = new OrbitViewItemControl();
+            ApplyImplicitOffsetAnimation(element);
+            return element;
+        }
+
+        protected override void PrepareContainerForItemOverride(DependencyObject element, object item)
+        {
+            if (_panel == null && ItemsPanelRoot != null)
+            {
+                _panel = ItemsPanelRoot as OrbitViewPanel;
+                _panel.ItemArranged += OrbitViewPanel_ItemArranged;
```

Comment:
Should we unsubscribe from this events before subscribe as it seems to me that this function is will repeat.